### ISSUE TITLE
A11Y: Focus border missing on gallery control

### ISSUE THREAD

**Issue by paul-geoghegan** [ubuntu/app-center#2161]
### Is there an existing issue for this?

- [x] I have searched the existing issues and found none matching what I'm reporting.

### Bug Description

Currently the control for the gallery is lacking a focus border and instead uses the default Flutter focus background which does  not meet contrast requirements.

### Steps to reproduce the behavior

1. Tab till you reach the gallery control
2. Notice the lack of a focus border

### Expected behavior

The control should use the same focus border used through out the app center.

### Ubuntu release

26.10 dev
